keyboards/ymdk/ymd60/info.json

```diff
@@ -0,0 +1,236 @@
+            {"matrix": [1, 6], "x": 97, "y": 16, "flags": 4},
+            {"matrix": [1, 7], "x": 112, "y": 16, "flags": 4},
+            {"matrix": [1, 8], "x": 127, "y": 16, "flags": 4},
+            {"matrix": [1, 9], "x": 142, "y": 16, "flags": 4},
+            {"matrix": [1, 10], "x": 157, "y": 16, "flags": 4},
+            {"matrix": [1, 11], "x": 172, "y": 16, "flags": 4},
+            {"matrix": [1, 12], "x": 187, "y": 16, "flags": 4},
+            {"matrix": [1, 13], "x": 205, "y": 16, "flags": 4},
+            {"matrix": [1, 14], "x": 224, "y": 16, "flags": 4},
+
+            {"matrix": [2, 0], "x": 6, "y": 32, "flags": 8},
+            {"matrix": [2, 1], "x": 26, "y": 32, "flags": 4},
+            {"matrix": [2, 2], "x": 41, "y": 32, "flags": 4},
+            {"matrix": [2, 3], "x": 56, "y": 32, "flags": 4},
+            {"matrix": [2, 4], "x": 71, "y": 32, "flags": 4},
+            {"matrix": [2, 5], "x": 86, "y": 32, "flags": 4},
+            {"matrix": [2, 6], "x": 101, "y": 32, "flags": 4},
+            {"matrix": [2, 7], "x": 116, "y": 32, "flags": 4},
+            {"matrix": [2, 8], "x": 131, "y": 32, "flags": 4},
+            {"matrix": [2, 9], "x": 146, "y": 32, "flags": 4},
+            {"matrix": [2, 10], "x": 161, "y": 32, "flags": 4},
+            {"matrix": [2, 11], "x": 175, "y": 32, "flags": 4},
+            {"matrix": [2, 12], "x": 200, "y": 32, "flags": 1},
+            {"matrix": [2, 14], "x": 224, "y": 32, "flags": 4},
+
+            {"matrix": [3, 0], "x": 9, "y": 48, "flags": 1},
+            {"matrix": [3, 1], "x": 34, "y": 48, "flags": 4},
+            {"matrix": [3, 2], "x": 49, "y": 48, "flags": 4},
+            {"matrix": [3, 3], "x": 63, "y": 48, "flags": 4},
+            {"matrix": [3, 4], "x": 78, "y": 48, "flags": 4},
+            {"matrix": [3, 5], "x": 93, "y": 48, "flags": 4},
+            {"matrix": [3, 6], "x": 108, "y": 48, "flags": 4},
+            {"matrix": [3, 7], "x": 123, "y": 48, "flags": 4},
+            {"matrix": [3, 8], "x": 138, "y": 48, "flags": 4},
+            {"matrix": [3, 9], "x": 153, "y": 48, "flags": 4},
+            {"matrix": [3, 10], "x": 168, "y": 48, "flags": 4},
+            {"matrix": [3, 11], "x": 189, "y": 48, "flags": 1},
+            {"matrix": [3, 12], "x": 209, "y": 48, "flags": 4},
+            {"matrix": [3, 14], "x": 224, "y": 48, "flags": 4},
+
+            {"matrix": [4, 0], "x": 2, "y": 64, "flags": 1},
+            {"matrix": [4, 1], "x": 21, "y": 64, "flags": 1},
+            {"matrix": [4, 2], "x": 39, "y": 64, "flags": 1},
+            {"matrix": [4, 5], "x": 95, "y": 64, "flags": 4},
+            {"matrix": [4, 9], "x": 151, "y": 64, "flags": 1},
+            {"matrix": [4, 10], "x": 170, "y": 64, "flags": 1},
+            {"matrix": [4, 11], "x": 194, "y": 64, "flags": 4},
+            {"matrix": [4, 12], "x": 209, "y": 64, "flags": 4},
+            {"matrix": [4, 13], "x": 224, "y": 64, "flags": 4},
```

Comment:
This RGB matrix data declares some LEDs with key matrix positions that don't correspond to keys actually defined in the layout (e.g., the `[4, 12]` and `[4, 13]` positions are not mentioned in the layout).  In total, there are 67 LEDs with `matrix` defined here, while the keyboard layout apparently has just 62 keys.  Looks like this data had been copied from some other keyboard, and now needs to be revised (including the LED coordinates).